File: MiniERP_UretimTakip/Models/UretimKaydi.cs
using System;

namespace MiniERP_UretimTakip.Models
{
    public class UretimKaydi
    {
        public int UretimID { get; set; }
        public int UrunID { get; set; }
        public int MakineID { get; set; }
        public int PersonelID { get; set; }
        public int UretimAdedi { get; set; }
        public DateTime UretimTarihi { get; set; }
        public string Aciklama { get; set; }
    }
}

File: client-mobile/NexusERP/app/src/main/java/com/tanerkaynar/nexuserp/data/model/UretimKaydi.kt
package com.tanerkaynar.nexuserp.data.model

data class UretimKaydi(
    val uretimid: Int = 0,
    val urunadi: String?,
    val makineadi: String?,
    val personeladi: String?,
    val uretimadedi: Int,
    val uretimtarihi: String?,
    val aciklama: String?
)

data class UretimRequest(
    val urunid: Int,
    val makineid: Int,
    val personelid: Int,
    val uretimadedi: Int,
    val aciklama: String
)